### PR TITLE
hometask4

### PR DESCRIPTION
**Pull request by zipic**
- [DEMO LINK](https://zipic.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://zipic.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,40 @@
...
+    <link rel="stylesheet" href="./css/style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__img" src="./images/imac.jpeg"
+      alt="Apple a1419 imac 27 Retina 5k Monoblock">
+      <div class="card__desciption">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
```
Comment (SanyaBratashchuk) [inline src/index.html:22]: use semantic tag

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,40 @@
...
+      <div class="card__code">Product code: 195434</div>
+      <div class="stars">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <p class="stars__review">Reviews: 5</p>
```
Comment (SanyaBratashchuk) [inline src/index.html:30]: It shouldn't be element of stars block

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,40 @@
...
+      </div>
+      <div class="card__price">
+        <div class="card__title">Price:</div>
+        <div class="card__sum">$2,199</div>
```
Comment (SanyaBratashchuk) [inline src/index.html:34]: use semantic tags

File: src/css/star.css
Comment (SanyaBratashchuk) [inline src/css/star.css:1]: rename file star => stars

File: src/css/style.css [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  --background--color-white: #fff;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.card {
```
Comment (SanyaBratashchuk) [inline src/css/style.css:15]: move styles related to card BEM block to the card.css file

File: src/css/card.css [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  margin-top: 16px;
+  margin-bottom: 24px;
+  align-items: center;
+}
+
+.card__review {
+  display: flex;
+  text-align: right;
+  font-style: normal;
+  font-weight: 400;
```
Comment (sTorba24) [inline src/css/card.css:49]: remove

File: src/css/card.css [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 16px;
+}
+
+.card__title {
+  display: inline-block;
+  color: #616070;
+  font-weight: 400;
```
Comment (sTorba24) [inline src/css/card.css:89]: remove

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,42 @@
...
+    <link rel="stylesheet" href="./css/style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__img" src="./images/imac.jpeg"
+      alt="Apple a1419 imac 27 Retina 5k Monoblock">
+      <article class="card__desciption">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </article>
```
Comment (sTorba24) [inline src/index.html:22]: use right semantic tags - `<p>` for example

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,42 @@
...
+      <article class="card__code">Product code: 195434</article>
```
Comment (sTorba24) [inline src/index.html:23]: here too

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,42 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__img" src="./images/imac.jpeg"
+      alt="Apple a1419 imac 27 Retina 5k Monoblock">
```
Comment (sTorba24) [inline src/index.html:19]:
```suggestion
      <img 
        class="card__img" 
        src="./images/imac.jpeg"
        alt="Apple a1419 imac 27 Retina 5k Monoblock"
      >
```

Comment (sTorba24) [reply src/index.html:19]: fix formatting

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Fix comments above

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**